Modules/Clients/Feature/Modules/RelationsExportImportTest.php

```
@@ -0,0 +1,118 @@
+        $content = $response->getContent();
+        $lines   = preg_split('/\r?\n/', mb_trim($content));
+        $this->assertGreaterThanOrEqual(1, count($lines));
+    }
+
+    #[Test]
+    #[Group('export')]
+    public function it_exports_relations_with_special_characters(): void
+    {
+        $this->markTestIncomplete();
+        /* Arrange */
+        $relations = Relation::factory()->for($this->company)->create(['name' => 'Rëlâtïon, "Test"', 'email' => 'special@example.com']);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$relations'. (undefined)

(UnusedLocalVariable)